Feature/storybook documentation

rishabhmalik759 opened the pull request:
Pull request to test qorus toolkit changes

Foxhoundn committed:
First storybook version

Foxhoundn committed:
Need to push iunfishnished work

Foxhoundn committed:
????????

Foxhoundn committed:
Wtf?

Foxhoundn committed:
Moved example to its own component

Foxhoundn committed:
Merge branch 'develop' into feature/storybook-documentation

Foxhoundn committed:
Documentation updates

Foxhoundn committed:
Merge branch 'develop' into feature/storybook-documentation

Foxhoundn committed:
Merge branch 'develop' into feature/storybook-documentation

Foxhoundn committed:
Merge branch 'develop' into feature/storybook-documentation

rishabhmalik759 committed:
Merge branch 'develop' into feature/storybook-documentation

rishabhmalik759 committed:
Merge branch 'feature/storybook-documentation' of https://github.com/qoretechnologies/qorus-toolkit into feature/storybook-documentation

rishabhmalik759 committed:
Changed from axios to node-fetch

rishabhmalik759 committed:
Fixed lint issues

rishabhmalik759 committed:
Changed initEndpoint to addEndpoint, changed files

rishabhmalik759 committed:
Changed initEndpoint to addEndpoint, updated files

rishabhmalik759 committed:
Fixed type bugs

rishabhmalik759 committed:
Updated reqore

rishabhmalik759 committed:
Updated test command

Foxhoundn committed:
Move dev dependencies to devDependencies

Foxhoundn committed:
Replace NPM with Yarn

Foxhoundn committed:
addEndpoint always returns something if it returns, otherwise it throws

Foxhoundn committed:
better messages on errors, check if endpoint exists before checking noauth

Foxhoundn committed:
Do not use AXIOS interfaces, we need to remove axios completely. Types generic requests

Foxhoundn committed:
captureStackTrace is only available in node or v8

Foxhoundn committed:
Demo is now working

Foxhoundn committed:
Nonsense rule

rishabhmalik759 committed:
Added request params to request

rishabhmalik759 committed:
Fixed lint issues

rishabhmalik759 committed:
Added documentation
- QorusDataProviders
- Qorus Request

rishabhmalik759 committed:
updated version

rishabhmalik759 committed:
Updated tsconfig

rishabhmalik759 committed:
Updated tsconfig

rishabhmalik759 committed:
Updated folder

rishabhmalik759 committed:
Updated imports

rishabhmalik759 committed:
Updated folder and tsconfig

rishabhmalik759 committed:
Updated unwanted imports

rishabhmalik759 committed:
Updated docs for
- QorusAuthenticator
- QorusOptions

rishabhmalik759 committed:
Added comment parser and libraries

rishabhmalik759 committed:
Updated test file name

rishabhmalik759 committed:
Updated sample file

rishabhmalik759 committed:
Fixed lint

Foxhoundn committed:
Documentation updates

Foxhoundn committed:
Updated documentation

rishabhmalik759 committed:
Added doc generator and updated typedoc config

rishabhmalik759 committed:
Updated test file name

rishabhmalik759 committed:
Updated doc script

rishabhmalik759 committed:
Added typedoc genration through module

rishabhmalik759 committed:
Added comment generator for classes

rishabhmalik759 committed:
Updated docs to support frontend

rishabhmalik759 committed:
Added docs for all methods

rishabhmalik759 committed:
Updated tests removed unwanted files

rishabhmalik759 committed:
Fixed type issues

rishabhmalik759 committed:
Fixed minor issues

rishabhmalik759 committed:
Fixed tests

rishabhmalik759 committed:
fixed lint issues

Foxhoundn committed:
Add tests file to gitignore, udpate reqore and storybook command, add AI generated comments

rishabhmalik759 committed:
Updated docs to use json object for rendering

rishabhmalik759 committed:
Fixed lint issues

rishabhmalik759 committed:
Updated build command and gitignore

rishabhmalik759 committed:
Updated build process

rishabhmalik759 committed:
Removed unwanted files

rishabhmalik759 committed:
Fixed type string conversion bug in docs

Foxhoundn committed:
Documentation updates, added Chromatic support.

Foxhoundn committed:
Fixed typo

Foxhoundn committed:
Add docs generation to storybook build

Foxhoundn committed:
Build docs in a different step

Foxhoundn committed:
Generating a TS file instead of a JSON one.

Foxhoundn committed:
ESlint....

Foxhoundn committed:
Chromatic is now working

Foxhoundn committed:
Fix ts erorr

rishabhmalik759 committed:
Updated docs :
- Added modified typescript-json-parser library
- Added isOptional for parameter types in classes

rishabhmalik759 committed:
Updated tests

rishabhmalik759 committed:
Merge branch 'feature/storybook-documentation' of https://github.com/qoretechnologies/qorus-toolkit into feature/storybook-documentation

rishabhmalik759 committed:
Updated version

rishabhmalik759 committed:
Updated lint-ignore

rishabhmalik759 committed:
Added docs for interfaces

Foxhoundn reviewed (changes requested):
why did you add
 "ts-node": "^10.9.1",
     "typedoc-json-parser": "^7.0.2"
to dependencies and not devDependencies ?
also what is that typedocParser for and why is it in git?

rishabhmalik759 committed:
Removed modified library

rishabhmalik759 committed:
Removed unwanted tests

rishabhmalik759 committed:
Fixed doc generation problem

rishabhmalik759 committed:
Removed unwanted tests

rishabhmalik759 committed:
Updated interface types parsing

rishabhmalik759 committed:
Fixed script

rishabhmalik759 committed:
Added docs for all methods

rishabhmalik759 committed:
Updated interface docs and fixed types

rishabhmalik759 committed:
Fixed  order of parsed types

rishabhmalik759 committed:
Fixed tests

rishabhmalik759 committed:
Fixed parameter types

rishabhmalik759 committed:
Added interface docs

rishabhmalik759 committed:
Removed old comments

rishabhmalik759 committed:
Updated release version

rishabhmalik759 committed:
Edited docs and comments

rishabhmalik759 committed:
Fixed tests

rishabhmalik759 committed:
Added docs for types and class properties

rishabhmalik759 committed:
Updated tests and examples

rishabhmalik759 committed:
Updated snapshots

rishabhmalik759 committed:
Fixed storybook errors

rishabhmalik759 committed:
Fixed storybook errors

rishabhmalik759 committed:
Fixed extra spaces bug in method definition

rishabhmalik759 committed:
Updated class property comments

rishabhmalik759 committed:
Updated docs for class properties and methods

rishabhmalik759 committed:
Updated interface docs

rishabhmalik759 committed:
Updated comments and docs

rishabhmalik759 committed:
Fixed parameter order

rishabhmalik759 committed:
Updated version

Foxhoundn committed:
Documentation updates

rishabhmalik759 committed:
Fixed interface type bug

rishabhmalik759 committed:
Added examples files to ts parser
- Added parser
- Updated file names
- Updated DocGenerator

rishabhmalik759 committed:
Fixed bugs in example converter

rishabhmalik759 committed:
Removed console logs

rishabhmalik759 committed:
Updated Property story names

rishabhmalik759 committed:
Updated examples

rishabhmalik759 committed:
Converted arrow method to normal method

rishabhmalik759 committed:
Updated codeExamples

rishabhmalik759 committed:
Converted codeExample.ts to json

rishabhmalik759 committed:
Fixed type bugs and updated docs

rishabhmalik759 committed:
Updated exports and tests

rishabhmalik759 committed:
Updated story name

rishabhmalik759 committed:
Updated release version

rishabhmalik759 committed:
Updated story name

rishabhmalik759 committed:
Moved type aliases to typings category

rishabhmalik759 committed:
Updated version

rishabhmalik759 committed:
Updated code example parser
- Modified parser to create a key with class-name.method-name

rishabhmalik759 committed:
Removed css files for typedoc

rishabhmalik759 committed:
Removed unwanted files

rishabhmalik759 committed:
Removed unwanted console logs

Foxhoundn committed:
Storybook Updates

Overview for all classes, links on all types & interfaces that should have links.

rishabhmalik759 committed:
Added examples for all methods

rishabhmalik759 committed:
Updated release version

rishabhmalik759 committed:
Merge branch 'feature/storybook-documentation' of https://github.com/qoretechnologies/qorus-toolkit into feature/storybook-documentation

rishabhmalik759 committed:
Fixed DocGenerator after merge

Foxhoundn committed:
Merge

rishabhmalik759 committed:
Added docs for all missing interfaces and type-aliases

rishabhmalik759 committed:
Fixed lint issues

rishabhmalik759 committed:
Updated eslint ignore

rishabhmalik759 committed:
Updated lint

rishabhmalik759 committed:
Fixed type issues

rishabhmalik759 committed:
Updated comments

Foxhoundn committed:
merge

rishabhmalik759 committed:
Updated class comments

rishabhmalik759 committed:
Merged changed

rishabhmalik759 committed:
Updated QorusDataProvider and QorusOptions

rishabhmalik759 committed:
Updated docs for classes, methods, interface and types

rishabhmalik759 committed:
Updated DocGenerator, fixed Record type, removed private methods, updated docs

rishabhmalik759 committed:
Updated release version

rishabhmalik759 committed:
Updated docs and missing methods

rishabhmalik759 committed:
Updated examples and fixed bugs

rishabhmalik759 committed:
Added missing docs, updated interface types to use Record

rishabhmalik759 committed:
Updated release version

rishabhmalik759 committed:
Updated type name

rishabhmalik759 committed:
Updated interfaces to use I suffix

rishabhmalik759 committed:
Added T prefix in front of types

rishabhmalik759 committed:
Fixed type issues

rishabhmalik759 committed:
Fixed types issues in QorusRequest

rishabhmalik759 committed:
Updated types and interfaces in documentation

rishabhmalik759 committed:
Fixed chromatic mentioned errors

rishabhmalik759 committed:
Fixed type issues

rishabhmalik759 committed:
Fixed chromatic errors

rishabhmalik759 committed:
Removed axios dependencies

rishabhmalik759 committed:
Remove unwanted dependencies, fixed test errors

rishabhmalik759 committed:
Fixed storybook issues

rishabhmalik759 committed:
Removed extra spaces

rishabhmalik759 committed:
Updated ci, fixed deepcan

rishabhmalik759 committed:
Updated interface names

Foxhoundn committed:
Documentation updates

Foxhoundn committed:
Remove console log

rishabhmalik759 committed:
Updated CI/CD

rishabhmalik759 committed:
Fixed deepscan issues

rishabhmalik759 committed:
Fixed remaining deepscan issues

rishabhmalik759 committed:
Fixed remaining deepscan issues

rishabhmalik759 committed:
Fixed remaining deepscan issues

rishabhmalik759 committed:
Updated interface and type names for QorusOptions

rishabhmalik759 committed:
Updated type from string to object for docs, added tests

rishabhmalik759 committed:
Removed unwanted console logs

rishabhmalik759 committed:
Updated types

rishabhmalik759 committed:
Fixed tests

rishabhmalik759 committed:
Added github action notification for discord

rishabhmalik759 committed:
Updated discord notification

rishabhmalik759 committed:
Updated discord notification, included steps

rishabhmalik759 committed:
Updated discord notification, edited notification

rishabhmalik759 committed:
Updated discord notification, testing output format

Foxhoundn committed:
Updated documentation to support new types, updated reqore

Foxhoundn committed:
Update Version

Foxhoundn committed:
Updated prettier error

Foxhoundn committed:
PRettier fixes

rishabhmalik759 committed:
Updated type for class property

Foxhoundn committed:
Updated documentation & reqore
